controller/src/main/java/io/pravega/controller/store/stream/PravegaTablesStreamMetadataStore.java

```diff
@@ -96,6 +96,21 @@
         this.executor = executor;
     }
 
+    @VisibleForTesting
+    PravegaTablesStreamMetadataStore(CuratorFramework curatorClient, ScheduledExecutorService executor,
```

Comment:
do we need this new constructor? can we reuse one of the existing constructors that takes segment helper for which we have a test mock that mocks the underlying store's behaviour.

Reply:
Yes, needed to create this constructor, because I had to mock getCachedData() on PravegaTablesStoreHelper. I tried mocking getEntry() on SegmentHelper mock, but it required metadataTableName which is not available in the test case and without the table name, just with the 'key' for getEntry set to "epoch-0" the mock method did not return the expected value.